contracts/example/tests/test.rs

```diff
@@ -52,15 +52,16 @@ fn setup_app<'a>(
 #[test]
 fn test_gmp_example() {
     let env = Env::default();
-
     env.mock_all_auths();
 
+    let user: Address = Address::generate(&env);
+
     // Setup source Axelar gateway
     let source_chain = String::from_str(&env, "source");
     let (source_gateway_client, source_gateway_id, _) = setup_gateway(&env);
     let (_source_gas_service_client, _source_gas_collector, source_gas_service_id) =
         setup_gas_service(&env);
-    let (source_app, source_app_id) = setup_app(&env, &source_gateway_id, &source_gas_service_id);
+    let (source_app, _source_app_id) = setup_app(&env, &source_gateway_id, &source_gas_service_id);
```

Comment:
```suggestion
    let source_app = setup_app(&env, &source_gateway_id, &source_gas_service_id);
```
don't return it if it isn't being used